### ISSUE TITLE
Stop core-activate event if a custom button-like element is clicked?

### ISSUE THREAD

**Issue by jakemac53** [Polymer/core-list#77]
Original report https://github.com/dart-lang/core-elements/issues/160.

Basically, if your core-list template has a custom element whose template contains a paper-button or something similar then when that button is clicked the core-activate event is always fired on the list item and there doesn't appear to be an easy way to stop it.

**Comment by firasdib** [comment]
Any solution to this? I still have not found a way of blocking this event which is a complete deal breaker for me. The core-list does everything else I want.

**Comment by DanielJoyce** [comment]
Yeah, core-list is so buggy. Same issue here. 9 times out of ten I just use a template repeat because I can easily get the handler behavior I need.
